src/Bundle/Resources/config/services/state.xml

```diff
@@ -95,15 +95,17 @@
             <tag name="sylius.state_provider" />
         </service>
 
-        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor">
+        <service id="Sylius\Resource\Doctrine\Common\State\PersistProcessor">
             <argument type="service" id="doctrine" />
             <tag name="sylius.state_processor" />
         </service>
+        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor" alias="Sylius\Resource\Doctrine\Common\State\PersistProcessor" />
 
-        <service id="Sylius\Component\Resource\Doctrine\Common\State\RemoveProcessor">
+        <service id="Sylius\Resource\Doctrine\Common\State\RemoveProcessor">
             <argument type="service" id="doctrine" />
             <tag name="sylius.state_processor" />
         </service>
+        <service id="Sylius\Component\Resource\Doctrine\Common\State\RemoveProcessor" alias="Sylius\Resource\Doctrine\Common\State\RemoveProcessor" />
```

Comment:
Is deprecation notice needed over here?

Reply:
thx, you have good eyes, I have to remove it too.